apps/course-authoring/src/renderer/pages/home/index.tsx

```diff
@@ -28,6 +28,22 @@ const Header = (
           Open
         </Btn>
       </div>
+      <div>
+        <Btn
```

Comment:
If we add the `invoke` method to the existing `window.electron` we can replace it with:
```tsx
        <Btn
          size="Sm"
          onClick={async () => {
            const messageFromMain = await window.electron.ipcRenderer.invoke(
              'start-scorm-export-process',
              { courseName: 'Test Course' }
            );

            console.log(messageFromMain);
          }}
        >
```